### ISSUE TITLE
primefaces 14.0.2 ScatterDataPoint class is missing

### ISSUE THREAD

**Issue by iGuv11** [primefaces/primefaces#12861]
### Describe the bug

Hello i am using primefaces 14.0.2 and primefaces extensions 14.0.2 with classifer as jakarta (4.1.0) in my jsf project. I want to draw linechart with data as ScatterDataPoint class but the is no import for this. After i searched it in internet i found that folder which has ScatterDataPoint.class is missing in my jar how do i fix it? Thanks for the answers
![image](https://github.com/user-attachments/assets/704224bd-03f0-4e53-ae08-c9c05abc3dd3)

### Expected behavior

Add linechart data as a ScatterDataPoint .

### PrimeFaces edition

None

### PrimeFaces version

14.0.2

### JSF implementation

All

### JSF version

4.1

### Java version

21.0.3

**Comment by tandraschko** [comment]
did you include the xdev dependency like stated in the docs?

@melloware AFAIR we added the includes the in the showcase; just not deployed yet, right?

**Comment by melloware** [comment]
Yes @iGuv11 go here to see the Scatter properly: https://quarkus-faces-melloware-8a6a34c1.koyeb.app/ui/chart/scatter.xhtml?jfwid=bbad5

Make sure you include:

```xml
<dependency>
    <groupId>software.xdev</groupId>
    <artifactId>chartjs-java-model</artifactId>
    <version>2.3.1</version>
</dependency>
```

@tandraschko yes the showcase is updated I believe just not deployed yet...

**Comment by tandraschko** [comment]
:/ we would really benefit from a auto-deployment somewhere, free and opensource

**Comment by iGuv11** [comment]
First of all, thank you very much, I was able to import ScatterDataPoint after adding the dependecy. However, when I added data with this data type in the graph I drew with a deprecated linechart, I thought it would look like the 1st photo, but I cannot draw the graph. I wonder how I should follow to make the point on the cursor in the deprecated model look like the 1st photo. In addition, I draw the graph with NumericPoint, but the data; It looks like the image in the 2nd photo. Since my graphic structure is very large, it will take me a long time to switch to the new structure.
![image](https://github.com/user-attachments/assets/46f0905e-0c60-40fe-bb3c-fb5c4aaf540f)
![image](https://github.com/user-attachments/assets/bd6e7725-b15c-492f-8e28-2ae1cde57f41)
